src/pricer/utils.ts

```diff
@@ -22,35 +23,19 @@ const tokenToApi: Record<string, ExchangeApiName[]> = {
   TKN: ['testExchange'],
 };
 
+const CACHE_EXPIRATION_TIME = 60_000;
 const INTERMEDIATE_CURRENCY = 'USD';
 
-type CoinBaseConstructorArgs = ConstructorParameters<typeof CoinBase>;
-type CoinCodexConstructorArgs = ConstructorParameters<typeof CoinCodex>;
-type CoinGeckoConstructorArgs = ConstructorParameters<typeof CoinGecko>;
-type ConstructorArgs =
-  | CoinBaseConstructorArgs
-  | CoinCodexConstructorArgs
-  | CoinGeckoConstructorArgs;
-
-const builders = new Map<
-  ExchangeApiName,
-  (args?: ConstructorArgs) => BaseExchangeApi
->();
-builders.set(
-  'coinBase',
-  (args: CoinBaseConstructorArgs = []) => new CoinBase(...args)
-);
-builders.set(
-  'coinCodex',
-  (args: CoinCodexConstructorArgs = []) => new CoinCodex(...args)
-);
+const builders = new Map<ExchangeApiName, () => ExchangeApi>();
```

Comment:
Do we want to cache the value of all the exchanges?

Reply:
I decided to apply to all of them but to share the same hard-coded expiration time. If required, we could eventually make it configurable, but for now this is not a need.